src/ui/WebView.cpp

```diff
@@ -171,6 +175,9 @@ namespace wfl::ui
     {
         auto const webContext = webkit_web_view_get_context(*this);
 
+        std::string configDir = Glib::get_user_config_dir();
+        std::string cssFilePath = configDir +  "/" + WFL_NAME + "/custom.css";
```

Comment:
Also auto

Reply:
Also, maybe we can rename the file as `web.css` in case we want to introduce a custom `app.css` for gtk as well.